.github/workflows/test-action.yml

```diff
@@ -0,0 +1,59 @@
+on: push
+name: Tests suite
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Common tests setup
+        uses: ./.github/actions/tests-common
+
+      - name: Setup dependencies
+        run: make docker-deps
+
+      - name: Execute tests
+        run: make docker-test
+
+  lint:
+    name: Automated linting
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Common tests setup
+        uses: ./.github/actions/tests-common
+
+      - name: Setup dependencies
+        run: make docker-deps
+
+      - run: make docker-lint
+
+  dialyzer:
+    name: Dialyzer
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Common tests setup
+        uses: ./.github/actions/tests-common
+
+      - name: Setup dependencies
+        run: make docker-deps
+
+      - run: make docker-dialyzer
```

Comment:
Before enabling this could you try to cache the dialyzer plt? GH actions are so slow...

Reply:
Had forgotten about caching, just tested it and it went from 11min to 1. Thanks!

Reply:
Great! For the first run of a new PR, in case the cache is not accessed/available maybe we could have something like:
```
    dialyzer: [
      plt_file: {:no_warn, "priv/plts/dialyzer.plt"}
    ]
```

Reply:
Done, thanks!

Reply:
Good to know it works even with plts in ignore. :rocket: